### ISSUE TITLE
Dataset Used

### ISSUE THREAD

**Issue by vrindaprabhu** [BenBBear/253-Final-Project#1]
Hi,

Did you use the same GRID corpus used in the paper? If not could you please share the link where the data can be downloaded from? 

Thanks.

**Comment by mattdawson** [comment]
Is this it?

http://spandh.dcs.shef.ac.uk/gridcorpus/

**Comment by voletiv** [comment]
Yes, the dataset used seems to be GRIDcorpus - since the paper being followed uses that, and the dict_size has been mentioned as 53 (same as GRID)
